Development-Guide/CAN-Filter-Guide/README.adoc

```diff
@@ -0,0 +1,60 @@
+== CAN Message Filter Guide
+----
+This is a basic guide to making a CAN message filter
+----
+
+==== Introduction
+The two main components of a filter are an ID and a Mask.
+The ID will be the unique identifier for the message you are looking for, and the Mask will determine which bits of the ID are important to sift through (this makes the proccess much more efficient and flexible). 
+
+IMPORTANT: A Mask must be able to adapt to future additions to a CAN network. It must be made *as tight as possible for longevity*.
+
+There is no set order to follow the steps, but to get started follow the steps below until you are comfortable.
+Another tip is to convert the hex numbers into binary numbers, determine the ID and Mask, and then convert those back into hex.
```

Comment:
Since hex and binary numbers aren't actually different, they should be referred to as formats. Try:

`...convert the numbers from hexadecimal format to binary format,...`